File: hackerrank/problem solving/algorithms/1 - warmup/1 - solve me first/SolveMeFirst.java
// https://www.hackerrank.com/challenges/solve-me-first/problem?isFullScreen=true

import java.util.Scanner;

public class SolveMeFirst {
    public static void main(String[] args) {
        var scan = new Scanner(System.in);
        int num1 = scan.nextInt();
        int num2 = scan.nextInt();
        System.out.println(solveMeFirst(num1, num2));
    }

        private static int solveMeFirst(final int a, final int b) {
            return a + b;
        }
}

File: hackerrank/problem solving/algorithms/1 - warmup/1 - solve me first/with morfo library/solve_me_first.go
// https://www.hackerrank.com/challenges/solve-me-first/problem?isFullScreen=true

package main

import (
	"fmt"
	"morfo/morfoio"
)

func main() {
	n1, n2 := morfoio.Read2[int, int]()
	fmt.Println(solveMeFirst(n1, n2))
}

// T: O(1)
// S: O(1) extra space
func solveMeFirst(n1, n2 int) int {
	return n1 + n2
}
